src/app/pages/news/9.4/05-azure-ad.md

```diff
@@ -1,19 +1,16 @@
 ## Azure Active Directory {#azure}
 
-Add more meta information to your process elements to ease finding or calling them.
+With Azure Active Directory, Axon Ivy is now able to support its first cloud-based Identity Provider. This means that your employees can simply use the well-known Microsoft login in order to access the Axon Ivy Platform.
 
-- __Tags__: Tag your process elements to easily find them in your projects. Some tags like `Deprecated`, `Connector` or `Demo` have a special meaning.
-- __Parameter Description__: Describe the parameters of start elements to give useful hints to the callers.
+- __Simple user management__: Maintain your users in one place, namely Azure Active Directory. Easily synchronize these users with Axon Ivy and make your business processes instantly accessible to the entire organization.
+
+- __User credentials__: Employees do not have to worry about additional user credentials. Access is simple and secure via the Microsoft login.
+
+- __OAuth 2.0__: With Microsoft login, the industry-standard OAuth 2.0 protocol is used for authorization. Consequently, Axon Ivy is no longer aware of these users' passwords and secure delegated access is granted.
```

Comment:
authroization -> authentication ?!

Reply:
> authroization -> authentication ?!

maybe both would be true..

Reply:
First, we need to authenticate so we know who is accessing the system.
Then, we have to find out what he is allowed to do - authorize.

Reply:
Thanks @phochstr for illuminating